moveit_ros/planning/planning_request_adapter_plugins/src/add_iterative_spline_parameterization.cpp

```diff
@@ -68,7 +68,10 @@ class AddIterativeSplineParameterization : public planning_request_adapter::Plan
       ROS_DEBUG("Running '%s'", getDescription().c_str());
       if (!time_param_.computeTimeStamps(*res.trajectory_, req.max_velocity_scaling_factor,
                                          req.max_acceleration_scaling_factor))
+      {
         ROS_WARN("Time parametrization for the solution path failed.");
```

Comment:
```suggestion
         ROS_ERROR("Time parametrization for the solution path failed.");
```

If we're going to make failure of time parameterisation a negative result (ie: by returning `false`) it becomes an error, so we should notify the user about that.

A warning would imply that it's something to take note of, but is not fatal.

Reply:
Done -- though personally I wouldn't consider a failed plan/time parameterization to be fatal; I often attempt to make plans that aren't necessarily expected to succeed when considering several plan options.  (Though this PR isn't the place to get into a larger discussion of log levels and I'll be consistent with the rest of MoveIt).